packages/api/tests/endpoints/test-rules.js

```diff
@@ -33,17 +48,36 @@ const assertions = require('../../lib/assertions');
 const { app } = require('../../app');
 
 const esIndex = randomString();
-const testRule = fakeRuleFactoryV2();
+const workflow = randomId('workflow-');
+const testRule = {
```

Comment:
I stopped using it because it was setting values for provider and collection and then causing the tests to fail when it tried looking up the provider and collection for the rule. I have since removed the beforeEach newRule logic and used `fakeRuleFactoryV2` for the tests that need it and then setting `provider` and `collection` to `undefined`. Let me know what you think:

https://github.com/nasa/cumulus/pull/2012/commits/29332bdf9a575edf5381cb1a905ba9c105fb24e4